src/test/parse-fail/obsolete-proc.rs

```diff
@@ -14,6 +14,7 @@
 
 fn foo(p: proc()) { } //~ ERROR `proc` is a reserved keyword
 
-fn bar() { proc() 1; }
+fn bar() { proc() 1; } //~ ERROR `proc` is a reserved keyword
+                       //~^ ERROR expected
 
```

Comment:
in this test we get a more informative error message because we are panicking a little bit later